### ISSUE TITLE
Group chat individualization

### ISSUE THREAD

**Issue by teamfridge** [MasterScrat/Chatistics#44]
This may be functionality I haven't figured out yet, but would be great to parse group chats and get stats per individual user of that single group chat. Am exploring with Whatsapp exports but can't find a way as yet.

**Comment by whatsdis** [comment]
yeah i was disappointed to learn FB group chats is not supported.

**Comment by MasterScrat** [comment]
@teamfridge not sure I understand. When you parse logs from a group chat, each message should have its own `senderName`, which allows you to perform analysis on a per-individual basis. You just have to group/filter the dataframes using this field.

@whatsdis the README does state Facebook Messenger group chats are not supported at the moment: https://github.com/MasterScrat/Chatistics#support-matrix. 
Did you see anywhere where it states the opposite?

**Comment by seinwave** [comment]
@MasterScrat —

I also ran into this issue. I only get one`senderName` in my groupchat's plot, and it is what's filed as `conversationWithName` in the .csv exports:

![plot](https://user-images.githubusercontent.com/8683877/72991942-e4cee180-3dc0-11ea-895e-083c3b5dabac.png)

![csv](https://user-images.githubusercontent.com/8683877/72991936-e26c8780-3dc0-11ea-9b44-8b1501e74287.png)


Also of note is that the `senderName` in my group chat csv appears to be splitting people up by time zone.

There's not just "Matt" — there's "AM - Matt" and "PM - Matt". 

Not sure if that's part of the issue or what.

None of this intended as a knock against your work — I think you've done an amazing job!

**Comment by whatsdis** [comment]
@MasterScrat oh my bad! seems like I missed it you were right.

**Comment by LiamRR** [comment]
Hey @MasterScrat 

> When you parse logs from a group chat, each message should have its own `senderName`, which allows you to perform analysis on a per-individual basis. You just have to group/filter the dataframes using this field.

When running `python visualizer.py breakdown` on the Whatsapp exported logs (_group chat_), I too am getting the chats grouped by `conversationWithName` instead of having the graph show individual users.  When using the `--filter-sender` filter, I'm not getting the breakdown image.  I'm using `visualize.py breakdown --platforms whatsapp --filter-sender "John Doe,Jane Doe"`, possibly passing it in wrong?
